unitTests/AreaChairConsoleMenuBar.test.js

```diff
@@ -111,7 +111,7 @@ describe('AreaChairConsoleMenuBar', () => {
       areaChairName: 'Senior_Program_Committee',
     }
     render(<AreaChairConsoleMenuBar {...props} />)
-    expect(baseMenuBarProps.propertiesAllowed).toHaveProperty('Reviewers')
+    expect(baseMenuBarProps.propertiesAllowed).toHaveProperty('reviewers')
```

Comment:
why is this lowercase now?

Reply:
I think this relates to the query search modal. All the properties usually use camelcase, but some components weren't formatted correctly. I made the change [here](https://github.com/openreview/openreview-web/pull/1987/commits/0791e8b2b57ea302e45ac0e717c98493255ac468)